spec/config/routes.pending.yml

```diff
@@ -808,7 +808,6 @@ MiqPolicyController:
 - exp_token_pressed
 - index
 - miq_event_field_changed
-- miq_policy_edit
```

Comment:
🎉 This test is no longer pending since I was able to fix this action with the correct identifier.